paasta_tools/cli/cmds/validate.py

```diff
@@ -317,13 +328,27 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
-def validate_tron(service_path):
+def validate_tron(service_path, verbose=False):
     soa_dir, service = path_to_soa_dir_service(service_path)
     returncode = True
 
     for cluster in list_tron_clusters(service, soa_dir):
         if not validate_tron_namespace(service, cluster, soa_dir):
             returncode = False
+        elif verbose:
+            service_config = load_tron_service_config_no_cache(service, cluster)
```

Comment:
we probably want to use the cached version of this (which iirc, is the same function name minus `_no_cache`